files/task_helper.rb

```diff
@@ -1,6 +1,8 @@
 require 'json'
 
 class TaskHelper
+  @@params = nil # rubocop:disable Style/ClassVars
```

Comment:
I think it's safe to just add this to `.rubocop.yml`. I trust codereview to catch misuse of class variables.